src/node_crypto.cc

```diff
@@ -5263,7 +5263,7 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
   THROW_AND_RETURN_IF_NOT_BUFFER(args[1], "Salt");
 
   pass = static_cast<char*>(node::Malloc(passlen));
-  if (pass == nullptr) {
+  if (pass == nullptr && passlen > 0) {
     FatalError("node::PBKDF2()", "Out of Memory");
   }
```

Comment:
One benefit to calling `node::Malloc()` is that you're sure to either get a null or valid pointer. In most places we probably initialize the allocation to `nullptr`, but in a way `node::Malloc()` could be seen as an additional safety precaution?